src/styles/components/_Logo.scss

```diff
@@ -11,32 +11,32 @@
 	width: 292px;
 	height: 56px;
 
-	&.Logo-red { background-image: url( "../images/logo-red.svg" ); }
-	&.Logo-white { background-image: url( "../images/logo-white.svg" ); }
+	&.HMLogo-red { background-image: url( "../images/logo-red.svg" ); }
```

Comment:
These can be unested / unoverloaded. The colour variant is enough to get away with.